pkg/cmd/root.go

```diff
@@ -57,10 +57,36 @@ var cmdTimes = &cobra.Command{
 	},
 }
 
+var cmdSwitch = &cobra.Command{
+	Use:   "switch",
+	Short: "Switch active volume for commands below (commit, log etc)",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			fmt.Println("Please specify a volume name.")
+			os.Exit(1)
+		}
```

Comment:
The Python version fails the command if you run it with the wrong number of arguments:
```
luke@tiny:~/gocode/src/github.com/clusterhq/dvol$ dvol switch foo bar
Wrong number of arguments.
```
Whereas this would proceed, ignoring all but the first, AIUI. Perhaps add a check for `len(args) > 1`?